docs/Readme.md

```diff
@@ -27,24 +28,36 @@
     [Heuristic](Learning-Environment-Design-Heuristic-Brains.md),
     [Learning](Learning-Environment-Design-Learning-Brains.md)
 * [Learning Environment Best Practices](Learning-Environment-Best-Practices.md)
-* [Using the Monitor](Feature-Monitor.md)
-* [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
-* [Using an Executable Environment](Learning-Environment-Executable.md)
-* [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
+
+### Advanced Usage
+  * [Using the Monitor](Feature-Monitor.md)
+  * [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
+  * [Using an Executable Environment](Learning-Environment-Executable.md)
+  * [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
 
 ## Training
 
 * [Training ML-Agents](Training-ML-Agents.md)
+* [Using TensorBoard to Observe Training](Using-Tensorboard.md)
 * [Training with Proximal Policy Optimization](Training-PPO.md)
 * [Training with Soft Actor-Critic](Training-SAC.md)
+
+### Advanced Training Methods
+
 * [Training with Curriculum Learning](Training-Curriculum-Learning.md)
 * [Training with Imitation Learning](Training-Imitation-Learning.md)
 * [Training with LSTM](Feature-Memory.md)
 * [Training Generalized Reinforcement Learning Agents](Training-Generalized-Reinforcement-Learning-Agents.md)
+* [Training Using Concurrent Unity Instances](Training-Using-Concurrent-Unity-Instances.md)
+
+### Cloud Training (Unsupported)
+Here are the cloud training set-up guides for Azure and AWS. We no longer use them ourselves and 
```

Comment:
this seems out of place for a table of contents with links.  I would suggest we do:

1) Create a new Document called "Cloud Training (Amazon Web Services & Microsoft Azure)
2) Put this content and the two links in this new document
3) Remove the ### Cloud Training (Unsupported) Section
4) Add a link to this new doc under advanced training

Reply:
Following conversation with @mmattar I suggest removing the section on Cloud Training and the pages on cloud training with AWS/Azure since we do not use them ourselves and hence it is difficult to test the accuracy of the content in those pages.

@unityjeffrey @mmattar are you on board with this suggestion?

Reply:
sounds good!